components/ProductList.js

```diff
@@ -0,0 +1,37 @@
+import useSWR from "swr";
+import ProductCard from "./ProductCard";
+
+const fetcher = (url) => fetch(url).then((response) => response.json());
+
+export default function ProductList() {
+  const { data, isLoading } = useSWR("/api/shoppinglist", fetcher);
+  console.log(data);
+  if (isLoading) {
+    return <h1>Loading...</h1>;
```

Comment:
A better approach is to treat `<h1>` as the “main heading” of the page or section and use `<h2>/<h3>` for secondary headings. LEt's use  `<p>` or `<span>` for temporary/error messages.
```suggestion
    return <p>Loading...</p>;
```